### PR TITLE
Destroy confirmation position

### LINKED ISSUE #1643: Destroy confirmation can appear offscreen
The destroy model confirmation is not fixed to the screen so it can appear off screen if you've scrolled.

### LINKED ISSUE #1651: Choose cloud notification has lost styles
The notification at the top of the choose cloud page has lost its styling (you need to have no saved credentials for it to appear).

### PR DESCRIPTION
**Pull request by huwshimi**
Fix the position of the model destroy confirmation popup to always be centred on the screen. Fixes #1643.
Fix the styling of the notifications on the choose-cloud screen. Fixes #1651.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the profile
- open a couple of models
- scroll down the page
- click destroy on the last model
- the popup should appear on the screen
- add a service and click deploy (make sure you have no credentials or hack choose-cloud.js to not return any)
- check that the notifications are spaced correctly and have a grey background

File: .gitignore [hunk 1]
```diff
@@ -32,6 +32,7 @@ pip-selfcheck.json
 *.pyc
 python-shelltoolbox
 releases
+share
```
Comment (huwshimi) [inline .gitignore]: Drive-by. This dir appears to be created whenever you make the GUI.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5051/
Test PASSed.

File: .gitignore (hunk 1 above)
Comment (anthonydillon) [inline .gitignore]: Thanks

**Comment by anthonydillon** [comment]
Code looks good :+1:

**Comment by kadams54** [comment]
QA is :ok: and code is :+1: - thanks @huwshimi!

**Comment by kadams54** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
